src/main/java/org/dependencytrack/tasks/OsvDownloadTask.java

```diff
@@ -39,25 +38,36 @@
 import java.util.Collections;
 import java.util.Date;
 import java.util.List;
+import java.util.Scanner;
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipInputStream;
 
 import static org.dependencytrack.model.ConfigPropertyConstants.VULNERABILITY_SOURCE_GOOGLE_OSV_ENABLED;
 import static org.dependencytrack.model.Severity.getSeverityByLevel;
-import static org.dependencytrack.util.VulnerabilityUtil.*;
+import static org.dependencytrack.util.VulnerabilityUtil.normalizedCvssV3Score;
+import static org.dependencytrack.util.VulnerabilityUtil.normalizedCvssV2Score;
 
 public class OsvDownloadTask implements LoggableSubscriber {
 
     private static final Logger LOGGER = Logger.getLogger(OsvDownloadTask.class);
 
     private static final String OSV_BASE_URL = "https://osv-vulnerabilities.storage.googleapis.com/";
+    public static final String OSV_CONFIG_GROUP = "osv-ecosystems";
```

Comment:
Please adhere to the existing naming scheme (group: `vuln-source`), see https://github.com/DependencyTrack/dependency-track/blob/7685a29143dbb23f6d63af20ca4845790d81023b/src/main/java/org/dependencytrack/model/ConfigPropertyConstants.java#L50-L51

Maybe instead of creating separate properties for every ecosystem, how do you feel about making this a single property? It could then be a string of enabled ecosystems, e.g.:
```
vuln-source.osv.ecosystems.enabled=Debian;Go;Maven
```